bundles/org.openhab.binding.tacmi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -173,4 +173,50 @@
 		<state readOnly="true"/>
 		<config-description-ref uri="channel-type:tacmi:schemaApiDefaults"/>
 	</channel-type>
+
+
+	<!-- JSON -->
+
+
+	<thing-type id="cmiJSON" extensible="schema-switch-ro,schema-numeric-ro,schema-state-ro">
```

Comment:
Please update the default translations (tacmi.properties). Use the i18n tool to generate them.